packages/expo-audio/ios/AudioRecordingRequester.swift

```diff
@@ -1,6 +1,9 @@
 import ExpoModulesCore
 
+private let selector = "request" + "RecordPermission:"
```

Comment:
i was kind of afraid that it can be optimized by swift as just `requestRecordPermission:`

tried this on release build of bare-expo

```
$ strings BareExpo.app/BareExpo | grep RecordPermission
RecordPermission:
requestRecordPermission:
RecordPermission:
requestRecordPermission:
```

maybe we should use some runtime string manipulation instead

Reply:
Ah, makes sense. Ok, I'll try something else.